### ISSUE TITLE
EqMac ready for iOS 14

### ISSUE THREAD

**Issue by tosbsas** [bitgapp/eqMac#868]
Is EqMac ready for Sonoma?
And is There still Development as there are very old bugs

**Comment by eelpout** [comment]
installed 14.0 yesterday. sort of works but you may have to go through and select sources and options multiple times within eqMac to get it to show up as a "virtual" output source within Sound Settings. Hope the developer can address this. 

sometimes the audio seems slightly distorted too, like the preamp is being overdriven on my DAC which is new. but I'm not convinced this isn't the Mac OS audio chain itself.

**Comment by RonvD** [comment]
Yes, same problem here. After a reboot I must close EqMac and start it again to get sound through, so not it goes to my normal headphone output and not the headphone output (EqMac)

**Comment by armis182** [comment]
Yesterday updated my mac mini m2 pro to Sonoma and I could not adjust sound volume on my Philips monitor which is connected via hdmi. Also eqMac lagging and not responding when opened. So I believe that there is issue with eqMac and new OS 14.0. and if that functions is important, than should wait to update OS.

**Comment by MetalMania31** [comment]
Same is happening to my Mac Mini M2.  UPdated to Sonoma MacoS 14.  I have to close EqMac and start it again for it to work.

**Comment by eelpout** [comment]
Audacity (v3.3.3) is also crashing too for me occasionally when opening mp3s, so there is something going on.

**Comment by doctorkb** [comment]
I'm having similar issues with my Mac Mini (Intel). I found that turning eqMac off and on (using the button in the app) was enough to reset the outputs in the OS.

This has occurred on every reboot as well as any time coming back from sleeping displays (I should note, my use of eqMac is to have a volume control on an HDMI output).

**Comment by beadex** [comment]
Same problems. When I connect to Bluetooth headphones, in MacOS 13 it automatic switch the input through. In MacOS 14 it's switched to the Bluetooth inputs.

**Comment by Fudal** [comment]
> Yesterday updated my mac mini m2 pro to Sonoma and I could not adjust sound volume on my Philips monitor which is connected via hdmi. Also eqMac lagging and not responding when opened. So I believe that there is issue with eqMac and new OS 14.0. and if that functions is important, than should wait to update OS.

Me too.

**Comment by doctorkb** [comment]
@SebastianEShelby - have you looked at this at all?

**Comment by SebastianEShelby** [comment]
> @SebastianEShelby - have you looked at this at all?

I'm not on IOS. if you're talking about https://github.com/bitgapp/eqMac/issues/875, looks similar to my issue https://github.com/bitgapp/eqMac/issues/870.

**Comment by doctorkb** [comment]
> > @SebastianEShelby - have you looked at this at all?
> 
> I'm not on IOS. if you're talking about #875, looks similar to my issue #870.

I suspect OP meant MacOS not IOS.  But yes, this does look like #870 as well.

Sorry, I tagged you as I thought you were one of the devs.

**Comment by MetalMania31** [comment]
Sonoma 14.1 seems to have made this behavior worse.
